docs/data-ai/ai/create-dataset.md

```diff
@@ -4,6 +4,385 @@ title: "Create a dataset"
+                           y_min_normalized=box["yMinNormalized"],
+                           x_max_normalized=box["xMaxNormalized"],
+                           y_max_normalized=box["yMaxNormalized"]
+                       )
+
+               file_ids.append(binary_id)
+
+       await data_client.add_binary_data_to_dataset_by_ids(
+           binary_ids=file_ids,
+           dataset_id=dataset_id
+       )
+       print("Added files to dataset.")
+       print("https://app.viam.com/data/datasets?id=" + dataset_id)
+
+       viam_client.close()
+
+   if __name__ == '__main__':
+       asyncio.run(main())
+   ```
+
+1. Run the script to upload the images and their metadata into a dataset in Viam app providing the following input:
+
+   ```sh {class="command-line" data-prompt="$" }
+   python upload_data.py -org-id <ORG-ID> -api-key <API-KEY> \
+      -api-key-id <API-KEY-ID> -machine-part-id <MACHINE-PART-ID> \
+      -location-id <LOCATION-ID> -dataset-name <NAME>
+   ```
+
+1. Continue to [Train a tflite machine learning model](/data-ai/ai/train-tflite/).
+
+{{% /expand%}}
+
+{{< gif webm_src="/how-tos/capture-images.webm" mp4_src="/how-tos/capture-images.mp4" alt="Configuring data management for a camera in the viam app" max-width="600px" class="aligncenter" >}}
+
+{{< table >}}
+{{% tablestep link="/services/data/" %}}
+**1. Enable the data management service**
+
+In the configuration pane for your configured camera component, find the **Data capture** section.
+Click **Add method**.
+
+When the **Create a data management service** prompt appears, click it to add the service to your machine.
+You can leave the default data manager settings.
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**2. Capture data**
+
+With the data management service configured on your machine, configure how the camera component captures data:
+
+In the **Data capture** panel of your camera's configuration, select `ReadImage` from the method selector.
+
+Set your desired capture frequency.
+For example, set it to `0.05` to capture an image every 20 seconds.
+
+Set the MIME type to your desired image format, for example `image/jpeg`.
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**3. Save to start capturing**
+
+Save the config.
+
+With cloud sync enabled, your machine automatically uploads captured data to the Viam app after a short delay.
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**4. View data in the Viam app**
+
+Click on the **...** menu of the camera component and click on **View captured data**.
+This takes you to the data tab.
+
+![View captured data option in the component menu](/get-started/quickstarts/collect-data/cam-capt-data.png)
+
+If you do not see images from your camera, try waiting a minute and refreshing the page to allow time for the images to be captured and then synced to the app at the interval you configured.
+
+If no data appears after the sync interval, check the **LOGS** tab for errors.
+
+{{% /tablestep %}}
```

Comment:
I wonder if we should get rid of steps 1-5 and replace it with a prerequisite that cover the "Capture at least 10 images of anything you want your machine to recognize."  alongside the tips and tricks? I don't feel strongly that we have to but I think the tips and tricks may get missed if someone comes here with data and scrolls past all of these to get to what they need